ArraysHomework/ArraysHomework.cs

```diff
@@ -10,39 +12,49 @@ public class ArraysHomework
         public void Arrays()
         {
             // Create an array of ints of size 100
-            
+            int[] myArray = new int[100];
             Assert.IsTrue(myArray.Length == 100);
 
             // take the array made above and fill it so these tests pass  
-            
+            myArray[0] = 100;
+            myArray[50] = 150;
+            myArray[68] = 168;
+
             Assert.IsTrue(myArray[0] == 100);
             Assert.IsTrue(myArray[50] == 150);
             Assert.IsTrue(myArray[68] == 168);
 
+
             // fill all of the values in to make the loop on line 28 pass as is.
             // Do not edit the loop that assert the passing case. Instead, add code above it to make it pass.
 
-            // hint you need some form of repetition.
+            //hint you need some form of repetition. //the integer named i is set to zero. add 1 to i only if i is less than the number of elements in the array.
+
+            for (int i = 0; i < myArray.Length; i++)
+            {
+                myArray[i] = 10;
+            }
+
             for (int i = 0; i < myArray.Length; i++)        // Don't edit from HERE
             {                                               //
                 Assert.IsTrue(myArray[i] == 10);            // 
             }                                               // to HERE.
 
 
+
+
             // what wrong with this array declaration? Fix it and make it compile
-            string stringArray = new string[5];
+            string[] stringArray = new string[5];
 
             // What about this one? Fix it and make it compile
-            bool[] boolArray = new bool[];
+            bool[] boolArray = new bool[3];
 
             // What about this one? Fix it and make it compile
-            bool[] boolArray2 = new [20];
+            bool[] boolArray2 = new bool[20];
 
             // Take this array and copy its contents into a new array 
             char[] copyMe = { 'a', 'b', 'c' };
-            char[] copiedArray = new char[3];
-
-            
+            char[] copiedArray = copyMe;
```

Comment:
This doesn't create a new array. `copyMe` and `copiedArray` are both references to the same array.